### PR TITLE
es2015ify

### PR DESCRIPTION
**Pull request by SamVerschueren**
Here it is :).

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -1,29 +1,27 @@
...
-w3counter(cli.input[0]).then(function (types) {
-	types.forEach(function (type, i) {
-		i++;
-		console.log(i + '. ' + type.item + ' (' + type.percent + ')');
-	});
+w3counter(cli.input[0]).then(types => {
+	let i = 1;
+
+	for (const type of types) {
+		console.log(`${i++}. ${type.item} (${type.percent})`);
```
Comment (SamVerschueren) [inline cli.js]: Do you want `i++` outside of the statement to make it clearer of what happens?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "w3counter",
   "version": "2.0.1",
-  "description": "Get ten most popular screen resolutions, browsers, operating system and countries",
+  "description": "Get ten most popular screen resolutions, browsers and operating systems.",
```
Comment (sindresorhus) [inline package.json]: remove the trailing `.`

File: cli.js [hunk 3]
```diff
@@ -1,29 +1,27 @@
...
-		'  $ w3counter country',
-		'  $ w3counter os',
-		'  $ w3counter res'
-	].join('\n')
-});
+const cli = meow(`
+	Usage
+	  $ w3counter <type>
+
+	Example
```
Comment (sindresorhus) [inline cli.js]: Examples

File: cli.js [hunk 4]
```diff
@@ -1,29 +1,27 @@
...
+	  $ w3counter browser
+	  $ w3counter os
+	  $ w3counter res
+`);
 
 if (!cli.input.length) {
 	console.error('Provide a type');
```
Comment (sindresorhus) [inline cli.js]: > Specify a type

File: lib/utils.js [hunk 5]
```diff
@@ -1,45 +1,41 @@
...
-	return types[key];
-});
+exports.types = Object.keys(types).map(x => types[x]);
 
-exports.load = function () {
-	return got('www.w3counter.com/globalstats.php')
-		.then(function (res) {
-			var $ = cheerio.load(res.body);
+exports.load = () => got('www.w3counter.com/globalstats.php')
+	.then(res => {
```
Comment (sindresorhus) [inline lib/utils.js]: Put it on line 13

**Comment by SamVerschueren** [comment]
Fixed

File: cli.js (hunk 1 above)
Comment (sindresorhus) [inline cli.js]: 👎

**Comment by sindresorhus** [comment]
Looks good :)

**Comment by sindresorhus** [comment]
@kevva Could you split out the CLI and then do a major release?

**Comment by SamVerschueren** [comment]
Thanks for the review Sindre (as always) :)!

**Comment by SamVerschueren** [comment]
ping @kevva for new release :)

**Comment by kevva** [comment]
Thanks for this and excuse my absence :). Great work and thanks for reviewing @sindresorhus!

**Comment by SamVerschueren** [comment]
Thanks @kevva! No problem!

**Comment by kevva** [comment]
Ops, published using Node.js 6...

**Comment by SamVerschueren** [comment]
I'd suggest using `np`, it tests the node version :).

**Pull request merged**